helloworld/notebooks/README.md

```diff
@@ -0,0 +1,100 @@
+# Transmogrify on Jupyter
+
+In this section we will look at how Transmogrify can be run within Scala notebooks on
+Jupyter.
+
+We are going to leverage [BeakerX](http://beakerx.com/) scala kernel for Jupyter
+
+## Setup BeakerX on Linux / Ubuntu
+
+Prequisities :
+
+* Apache Maven
+* Python 3
+* JDK 8 (JDK 10 or above can cause issues with Transmogrify)
+
+Installation using pip
+
+```$xslt
+sudo pip install beakerx
```

Comment:
does this have to be sudo? (just curios)

Reply:
sometimes pip install fails because of permissions issue